src/App/Pages/Settings/AutofillServicesPage.xaml.cs

```diff
@@ -3,34 +3,33 @@
 
 namespace Bit.App.Pages
 {
-    public partial class AccessibilityServicePage : BaseContentPage
+    public partial class AutofillServicesPage : BaseContentPage
     {
-        private readonly AccessibilityServicePageViewModel _vm;
+        private readonly AutofillServicesPageViewModel _vm;
         private readonly SettingsPage _settingsPage;
         private DateTime? _timerStarted = null;
         private TimeSpan _timerMaxLength = TimeSpan.FromMinutes(5);
 
-        public AccessibilityServicePage(SettingsPage settingsPage)
+        public AutofillServicesPage(SettingsPage settingsPage)
         {
             InitializeComponent();
-            _vm = BindingContext as AccessibilityServicePageViewModel;
+            _vm = BindingContext as AutofillServicesPageViewModel;
             _vm.Page = this;
             _settingsPage = settingsPage;
         }
 
-        protected override void OnAppearing()
+        protected async override void OnAppearing()
         {
+            await _vm.InitAsync();
             _vm.UpdateEnabled();
-            _vm.UpdatePermitted();
             _timerStarted = DateTime.UtcNow;
-            Device.StartTimer(new TimeSpan(0, 0, 3), () =>
+            Device.StartTimer(new TimeSpan(0, 0, 0,0,500), () =>
```

Comment:
Please add some spaces in-between the numbers and commas in this constructor call.